src/framework/components/scroll-view/data.js

```diff
@@ -1,6 +1,10 @@
+import { Vec2 } from '../../../math/vec2.js';
+
 class ScrollViewComponentData {
     constructor() {
         this.enabled = true;
+        this.useMouseWheel = true;
```

Comment:
Open for discussion if this should be default `true` or not.

Reply:
I'd say that's 100% reasonable for it to default to true. 😄